src/components/v5/shared/MembersAvatars/MembersAvatars.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Id } from '@colony/colony-js';
+import { COLONY_TOTAL_BALANCE_DOMAIN_ID } from '~constants';
+import {
+  calculateLastSliceIndex,
+  calculateRemainingItems,
+} from '~utils/avatars';
+import { notNull } from '~utils/arrays';
+import UserAvatar from '~v5/shared/UserAvatar';
+import { MembersAvatarsProps } from './types';
+import { Watcher } from '~types';
+
+const displayName =
+  'v5.common.ActionSidebar.partials.motions.Motion.steps.OutcomeStep.partials.MembersAvatars';
+
+function MembersAvatars<TValue extends Watcher>({
+  items,
+  currentDomainId = COLONY_TOTAL_BALANCE_DOMAIN_ID,
+  maxAvatars = 4,
+  className,
+}: MembersAvatarsProps<TValue>): JSX.Element {
+  const remainingAvatarsCount = calculateRemainingItems(
+    maxAvatars,
+    items ?? [],
+    false,
+  );
+
+  return (
+    <div className={className}>
+      {(currentDomainId === Id.RootDomain ||
+        currentDomainId === COLONY_TOTAL_BALANCE_DOMAIN_ID) && (
+        <ul className="flex relative">
+          {items
+            .slice(0, calculateLastSliceIndex(maxAvatars, items, false))
+            .filter(notNull)
```

Comment:
This shouldn't be needed here. You should get an array with items that are in the right format